flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/streaming/util/TestStreamEnvironment.java

```diff
@@ -75,36 +81,28 @@ public static void setAsContext(
                     TestStreamEnvironment env =
                             new TestStreamEnvironment(
                                     miniCluster, parallelism, jarFiles, classpaths);
-                    randomize(conf);
+                    if (RANDOMIZE_CHECKPOINTING_CONFIG) {
+                        randomize(
+                                conf, ExecutionCheckpointingOptions.ENABLE_UNALIGNED, true, false);
+                        randomize(
+                                conf,
+                                ExecutionCheckpointingOptions.ALIGNMENT_TIMEOUT,
+                                Duration.ofSeconds(0),
+                                Duration.ofMillis(100),
+                                Duration.ofSeconds(2));
+                    }
+                    if (STATE_CHANGE_LOG_CONFIG.equals(STATE_CHANGE_LOG_CONFIG_ON)) {
+                        conf.set(CheckpointingOptions.ENABLE_STATE_CHANGE_LOG, true);
+                    } else if (STATE_CHANGE_LOG_CONFIG.equals(STATE_CHANGE_LOG_CONFIG_RAND)) {
```

Comment:
nit: `trim` + `equalsIgnoreCase`?

Reply:
@rkhachatryan Thanks for the review! I have applied the comment in another commit. Would you mind taking another look? Thanks.

Reply:
@rkhachatryan IMHO, this PR is to config the changelog state backend in tests. It might be better to merge them separately and then enable the randomization.

Reply:
Thanks. Sure!